src/widgetastic_patternfly/__init__.py

```diff
@@ -616,26 +616,33 @@ def __repr__(self):
         return '{}({!r})'.format(type(self).__name__, self.locator)
 
 
-class Tab(View, ClickableMixin):
+class Tab(View):
     """Represents the Tab widget.
 
     Selects itself automatically when any child widget gets accessed, ensuring that the widget is
     visible.
+
+    You can specify your own ``ROOT`` attribute on the class.
```

Comment:
I'm not sure about CF, but for sat specifying ROOT will break `TAB_LOCATOR` so tab locator should be specified as well.
Please take a look at your UI, but for us tab link and actual tab content are located in different parts of DOM, so `TAB_LOCATOR` should not rely on ROOT. If that's not the case for CF - then your solution is ok and we'll just provide our specific locators, but if that's the case for CF as well - then the solution is not universal enough and we'd rather think of other options.

Reply:
I think that when I use parent browser instead of browser, then that should work